indico/modules/categories/serialize.py

```diff
@@ -31,21 +31,23 @@
 from indico.web.flask.util import url_for
 
 
-def serialize_category_ical(category, user, event_filter):
+def serialize_category_ical(category_ids, user, event_filter=True, event_filter_fn=None, update_query=None):
     """Export the events in a category to iCal
 
-    :param category: The category to export
+    :param category_ids: Category IDs to export
     :param user: The user who needs to be able to access the events
     :param event_filter: A SQLalchemy criterion to restrict which
                          events will be returned.  Usually something
                          involving the start/end date of the event.
+    :param event_filter_fn: Event filter function to use when called from api.py
+    :param update_query: A query to update with when called from api.py
```

Comment:
That docstring is not very good. You are explaining how you currently use it, not what the parameter does.

> A callable that can update the query used to retrieve the events. Must return the updated query object.

That'd be clearer